float32 gripperPos
float32 gripperVel